src/components/navigation/sidebar.tsx

```diff
@@ -18,6 +19,7 @@ export default function Sidebar() {
           </div>
         </div>
       </div>
+      <FeedbackButton></FeedbackButton>
```

Comment:
What do you think about adding this on the layout instead of the sidebar? This way the fab icon could stay there even if the menu is not present.

Not necessary to change tho

Reply:
It was there,  on layout the window is not available,  think its because layout is server side render